app/assets/stylesheets/base/_base.scss

```diff
@@ -361,7 +377,7 @@
   .card-copy {
     font-size: 0.9em;
     line-height: 1.5em;
-    padding: ($base-spacing / 2);
+    padding: ($base-spacing / 2) ($base-spacing / 2);
```

Comment:
Shorthand form for property `padding` should be written more concisely as `$base-spacing / 2` instead of `$base-spacing / 2 $base-spacing / 2`